### ISSUE TITLE
Forced language setting via variable

### ISSUE THREAD

**Issue by SAOPP** [tarampampam/error-pages#91]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the problem to be solved

Hi!

Now the service can automatically determine the locale settings and install the language, I have a question, is it possible to set the desired language through a variable to turn off this check?

For example, I need only English, how can I do it now? Thank u in advance!

### Suggest a solution

Add environment variable for set up language of pages.

**Comment by tarampampam** [comment]
Hi @SAOPP!

Could you please explain what's the reason for your request? Automatic localization based on the browser locale (and nothing more), it's like a small piece of sugar for the end-users :) Maybe you have some problems with it?

**Comment by SAOPP** [comment]
Okay, why I have an english locale but I got russian pages? I think automatically detect it's not really cool, and I talked about variable for native setting this up.

**Comment by tarampampam** [comment]
Ok 👌, I will implement a flag for this a little bit later, ok?

**Comment by SAOPP** [comment]
Np homie :smiley: I still initially tried to understand whether it would be difficult to implement, thank u very much. :+1:

**Comment by tarampampam** [comment]
Could you please test changes in #94? Docker image (just for a test) `tarampampam/error-pages:toogle-l10n`

**Comment by tarampampam** [comment]
Ping @SAOPP

**Comment by tarampampam** [comment]
@SAOPP are you ok?

**Comment by SAOPP** [comment]
Yes, everything works, the language is ignored if you put non-English higher in the list in browser. :+1: 

Btw, I would like to apologize for the fact that when I wrote this issue, I didn't keep track of the fact that for some reason my 'non-English' language was higher than English language is Chrome, which is why I actually wrote this issue - because in the list of languages in It's not obvious to Chrome that it's from this list that the language is defined as the main language. I'm not sure if I explained it clearly enough, let me know.
